docs/setup_git.md

```diff
@@ -198,6 +198,60 @@ nothing to commit, working tree clean
 ```
 </div>
 
+## GitHub Authentication
+There are two ways to connect to GitHub: SSH Keys and GitHub Personal Access Tokens Keys.  An SSH Key is a special file that you can use to connect to remote terminals.  A Personal Access Token works like a separate password used just for GitHub.
+
+### SSH Keys (recommended)
+Follow the GitHub [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh) tutorial.
```

Comment:
Students might think they need to go through all the links, we can ask them to go through [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).